app/embedder.py

```diff
@@ -116,4 +123,15 @@ def _buildEmbedding(self, seq: str) -> np.array:
                 tf.compat.v1.tables_initializer()])
             out = sess.run(seq_output)[0][:len(tokens), :]
 
-            return out
+        return out
+
+
+@embed_bp.route('/embed', methods=['POST'])
+def make_embed():
```

Comment:
why the make_ prefix? Is this some convention?

Reply:
nope, fixed